docs/ingestion/ingestion-spec.md

```diff
@@ -226,7 +226,7 @@ Dimension objects can have the following components:
 | type | Either `string`, `long`, `float`, `double`, or `json`. | `string` |
 | name | The name of the dimension. This will be used as the field name to read from input records, as well as the column name stored in generated segments.<br /><br />Note that you can use a [`transformSpec`](#transformspec) if you want to rename columns during ingestion time. | none (required) |
 | createBitmapIndex | For `string` typed dimensions, whether or not bitmap indexes should be created for the column in generated segments. Creating a bitmap index requires more storage, but speeds up certain kinds of filtering (especially equality and prefix filtering). Only supported for `string` typed dimensions. | `true` |
-| multiValueHandling | Specify the type of handling for [multi-value fields](../querying/multi-value-dimensions.md). Possible values are `sorted_array`, `sorted_set`, and `array`. `sorted_array` and `sorted_set` order the array upon ingestion. `sorted_set` removes duplicates. `array` ingests data as-is | `sorted_array` |
+| multiValueHandling | Specify the type of handling for [multi-value fields](../querying/multi-value-dimensions.md). Possible values are `sorted_array`, `sorted_set`, and `array`. Both `sorted_array` and `sorted_set` order the array upon ingestion. For string typed dimensions, `sorted_set` removes duplicates. For string typed dimensions, `array` ingests data as is. | `sorted_array` |
```

Comment:
Good call changing this doc!

Since this is ignored for non-strings, IMO the doc should be even stronger, and explicitly say it's ignored for types other than `string`. How about:

```suggestion
| multiValueHandling | For `string` typed dimensions, specifies the type of handling for [multi-value fields](../querying/multi-value-dimensions.md). Possible values are `array` (ingest string arrays as-is), `sorted_array` (sort string arrays during ingestion), and `sorted_set` (sort and deduplicate string arrays during ingestion). This parameter is ignored for types other than `string`. | `sorted_array` |
```

Reply:
thank you for your suggestion! i will commit the suggestion.